app/src/main/java/ch/epfl/favo/viewmodel/FavorViewModel.java

```diff
@@ -276,6 +278,20 @@ public void setFavorValue(Favor favor) {
     return observedFavor;
   }
 
+
+  @Override
+  public LiveData<User> getObservedUser() {
```

Comment:
Should change name to getCurrentObservedUser() or getOwnUser().
On a side note implementing an observer makes it slower to load for the views. We should only use livedata if the user data is changing often, which I guess it isn't. Maybe it's easier to just UserUtil....findUser(currentUserId) every time we need the user. Up to us to decide!

Reply:
you are right, deleted it